xcode/Subconscious/Shared/Components/Detail/UserProfileDetailView.swift

```diff
@@ -289,7 +293,18 @@ struct UserProfileDetailModel: ModelProtocol {
             model.user = content.profile
             model.statistics = content.statistics
             model.recentEntries = content.recentEntries
-            model.following = content.following
+            model.following = content.following.map { follow in
+                if let petname = follow.user.address.petname,
+                   model.pendingFollows.contains(petname),
+                   follow.resolutionStatus != .resolved {
+                    
+                    var user = follow
+                    user.resolutionStatus = .pending
+                    return user
+                }
+                
+                return follow
+            }
```

Comment:
I am not in love with this, but the model is the right place to track pending follows imo.

I experimented with tracking them in the AddressBookService but that introduces some nuanced sequenced update problems, since the model ultimately drives everything and needs access to the pending list to render the UI.

I think we might want to move this up to an app-level concern ultimately? All these background processes are owned the by profile view which is discarded if we pop the navigation stack. Meaning that if I follow a user, they're pending resolution and I go back then forward again the user will not longer be "pending".